package.json

```diff
@@ -14,6 +14,7 @@
     "express": "^4.16.3",
     "mysql": "^2.16.0",
     "mysql2": "^1.6.1",
-    "sequelize": "^4.38.0"
+    "sequelize": "^4.38.0",
+    "tedious": "^2.6.4"
```

Comment:
why is this dependency installed and where is it used?

Reply:
at the time of using mssql we were asked to have the library installed to make the connection